tests/unit/plugins/module_utils/test_oidc_auth.py

```
@@ -0,0 +1,208 @@
+
+__metaclass__ = type
+
+import io
+import json
+from unittest.mock import patch, MagicMock
+from urllib.error import HTTPError
+
+from plugins.module_utils.oidc_auth import discover_oidc_token_endpoint, oidc_password_grant
+
+
+def _fake_http_response(data):
```

Ruff (0.16.0)

[warning] Missing return type annotation for private function `_fake_http_response`

(ANN202)